src/nvhttp.cpp

```diff
@@ -16,6 +16,7 @@
 #include <boost/property_tree/json_parser.hpp>
 #include <boost/property_tree/ptree.hpp>
 #include <boost/property_tree/xml_parser.hpp>
+#include <nlohmann/json.hpp>
```

Comment:
redundant include

Reply:
It's not clear to me where else this is included?